examples/tensorflow-tabular/sim.py

```diff
@@ -0,0 +1,288 @@
+    train_dataset = Dataset.from_dict(train_dataset)
+
+    # Extract the test data and set it as Dataset format
+    test_dataset = client_dataset_splits["test"]
+    test_dataset_features = test_dataset.remove_columns([col for col in test_dataset.column_names if col not in RELEVANT_FEATURES])
+    test_dataset_labels = test_dataset.remove_columns([col for col in test_dataset.column_names if col not in [LABEL_NAME]])
+    test_dataset = {"features": [list(row.values()) for row in test_dataset_features], "labels": [list(row.values()) for row in test_dataset_labels]}
+    test_dataset = Dataset.from_dict(test_dataset)
+
+    # Append the preprocessed FederatedDataset
+    fds_preprocessed.append(train_dataset)
+
+    # Concatenate the test partitions of each client into the centralized_testset
+    if centralized_testset is None:
+        centralized_testset = test_dataset
+    else:
+        centralized_testset = concatenate_datasets([centralized_testset, test_dataset])
+
+# Convert centralized dataset to tf tensor format
+centralized_testset = centralized_testset.to_tf_dataset(
+    columns="features", label_cols="labels", batch_size=64
+)
+
+# Create FedAvg strategy
+strategy = fl.server.strategy.FedAvg(
+    fraction_fit=1.0,  # Sample 100% of available clients for training
+    fraction_evaluate=0.5,  # Sample 50% of available clients for evaluation
+    min_fit_clients=2,  # Never sample less than 2 clients for training
+    min_evaluate_clients=2,  # Never sample less than 2 clients for evaluation
+    min_available_clients=int(
+        NUM_CLIENTS * 0.75
+    ),  # Wait until at least 75 clients are available
+    evaluate_metrics_aggregation_fn=weighted_average,  # aggregates federated metrics
+    evaluate_fn=get_evaluate_fn(centralized_testset),  # global evaluation function
+)
+
+
+# ClientApp for Flower-Next
+client = fl.client.ClientApp(
+    client_fn=get_client_fn(fds_preprocessed),
+)
+
+# ServerApp for Flower-Next
+server = fl.server.ServerApp(
+    config=fl.server.ServerConfig(num_rounds=NUM_ROUNDS),
+    strategy=strategy,
+)
+
+
+def main() -> None:
+    # Parse input arguments
+    args = parser.parse_args()
+
+    # With a dictionary, you tell Flower's VirtualClientEngine that each
+    # client needs exclusive access to these many resources in order to run
+    client_resources = {
+        "num_cpus": args.num_cpus,
+        "num_gpus": args.num_gpus,
+    }
+
+    # Start simulation
+    fl.simulation.start_simulation(
+    client_fn=get_client_fn(fds_preprocessed),
+    num_clients=NUM_CLIENTS,
+    config=fl.server.ServerConfig(num_rounds=NUM_ROUNDS),
+    strategy=strategy,
+    client_resources=client_resources,
+    actor_kwargs={
+        "on_actor_init_fn": enable_tf_gpu_growth  # Enable GPU growth upon actor init.
+    },
+    )
+
+
+if __name__ == "__main__":
+    # Enable GPU growth in your main process
+    enable_tf_gpu_growth()
+    main()
```

Comment:
we are gradually moving to a "Flower Next"-only mode. Therefore i think it would be better to exclude this (it's the old way of working with the non-FlowerNext simulation engine).